frontend/src/Components/BusinessDetails/BusinessDetails.jsx

```diff
@@ -89,8 +119,36 @@ const BusinessDetails = () => {
             </Modal>
           )}
           <div className="business-main-content">
-            <div className="business-section business-overview">
-              <BusinessOverview overview={business.overview} />
+            <div className="business-section business-map">
+              <APIProvider apiKey={apiKey}>
+                <Map
+                  defaultZoom={20}
+                  defaultCenter={parseLocation(business.location)}
+                  mapId={"218a557688af104"}
```

Comment:
This looks like it should be added to your .env file.

Reply:
Missed that! Thanks, will do